x/auth/keeper/grpc_query_test.go

```diff
@@ -191,3 +193,152 @@ func (suite *KeeperTestSuite) TestGRPCQueryParameters() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestBech32Prefix(){
+	suite.Run("TestBech32Prefix", func() {
+		suite.SetupTest() // reset
+		ctx := sdk.WrapSDKContext(suite.ctx)
+		req := &types.Bech32PrefixRequest{}
+		res, err := suite.queryClient.Bech32Prefix(ctx, req)
+		suite.Require().NoError(err)
+		suite.Require().NotNil(res)
+		suite.Require().Equal(res.Bech32Prefix,suite.app.AccountKeeper.GetBech32Prefix())
```

Comment:
```suggestion
		suite.Require().Equal(res.Bech32Prefix, "cosmos")
```

I think it's maybe better to test against something that's not the AccountKeeper field. Can be "cosmos", or can be sdk.Bech32Prefix too

Reply:
Done!